Makefile

```diff
@@ -113,23 +113,10 @@ pylint_report.txt: ${PYSOURCES}
 diff_pylint_report: pylint_report.txt
 	diff-quality --violations=pylint pylint_report.txt
 
-.coverage: $(PYSOURCES) all
-	export COVERAGE_PROCESS_START=${mkfile_dir}.coveragerc; \
-	       cd ${CWL}; ./run_test.sh RUNNER=cwltool
-	coverage run setup.py test
-	coverage combine ${CWL} ${CWL}/draft-3/ ./
+.coverage: tests
 
-coverage.xml: .coverage
-	python-coverage xml
-
-coverage.html: htmlcov/index.html
-
-htmlcov/index.html: .coverage
-	python-coverage html
```

Comment:
can we keep the HTML report?

Reply:
In retrospect, we can also keep the xml report